examples/online_serving/chart-helm/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   # -- Image tag
   tag: "latest"
   # -- Container launch command
-  command: ["vllm", "serve", "/data/", "--served-model-name", "opt-125m", "--dtype", "bfloat16", "--host", "0.0.0.0", "--port", "8000"]
+  command: ["vllm", "serve", "/data/", "--served-model-name", "opt-125m", "--dtype", "float32", "--block-size", "16", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
this need to be reverted i assume

Reply:
This is a curious issue that I have seen on CI where cpu build fails on that dtype.  I look around and see a similar issue which suggests switching to `float32` https://github.com/vllm-project/vllm/issues/11327#issuecomment-2774440315.  Let me try to reproduce it to get the full log.

Reply:
Here is the full server log when serving `vllm serve facebook/opt-125m` on the CPU docker image https://paste.sh/rDYEmrB9#9UuSY3jL6pHBE4MVy_lUDLle.  The change you see here is only to make it work on CI, let me cut an issue.